server/build.gradle.kts

```diff
@@ -57,8 +43,7 @@ dependencies {
     implementation("com.github.inorichi.injekt:injekt-core:65b0440")
     implementation("com.squareup.okhttp3:okhttp:4.9.1")
     implementation("io.reactivex:rxjava:1.3.8")
-    implementation("org.jsoup:jsoup:1.14.1")
-    implementation("com.google.code.gson:gson:2.8.7")
+    implementation("com.google.code.gson:gson:2.8.8")
```

Comment:
same for gson, don't remove jsoup from here, it's for reference...

Reply:
Duplicating it isn't a good idea, mismathing versions and all that. Maybe once we move to version catalogues or buildSrc version management I can re-add it?

Reply:
just don't do it for now...
I promise to be careful to not mismath.

Reply:
👌